src/plugins/vuex-store/modules/reviews/kylie.js

```diff
@@ -0,0 +1,97 @@
+         * Let's get started, shall we?
+         *
+         * First, please fill in the 'personal' object below with your data
+         * Guide:
+         *        title      - your honorific or name title
+         *                     examples: 'Mr.', 'Ms.', 'Dr.', 'Engr.'
+         *                     this is OPTIONAL (leave empty if you want to stay stupid like me)
+         *
+         *        firstName  - your first name in proper case
+         *                     examples: 'John', 'Mary', 'Bill', 'Anne Billy'
+         *                     this is REQUIRED (or I will call you 'Poop', I'm not kidding)
+         *
+         *        middleName - your middle name in proper case
+         *                     examples: 'Delos', 'Reyes', 'Delos Reyes'
+         *                     this is OPTIONAL (no need to check your birth certificate)
+         *
+         *        lastName   - your last name or surname in proper case
+         *                     examples: 'Doe', 'Dela Cruz'
+         *                     this is REQUIRED (sorry, there's really a need to check your birth certificate)
+         *
+         *        nameSuffix - your name suffix
+         *                     examples: 'Jr.', 'III'
+         *                     this is OPTIONAL (leave empty if you don't have this extra baggage in your name)
+         *
+         */
+
+        // this is that 'personal' object I mentioned above:
+        personal: {
+            title     : 'Mr.',
+            firstName : 'Kylie',
+            middleName: 'Gabalfin',
+            lastName  : 'Magistrado',
+            nameSuffix: '',
+        },
+        // ===================================================================================================
+
+
+        /**
+         * Next, please fill in the 'review' object below with your review on me.
+         * Guide:
+         *        as         - you're reviewing me as my _____?
+         *                     examples: 'friend', 'student', 'mentee', 'client', 'co-worker', 'boss'
+         *                     this is an ARRAY, so you can assign one or more values
+         *                     this is also REQUIRED (or I will tell Dr. Strange to delete you in my memory)
+         *
+         *        rating     - a number from 1 to 5
+         *                     examples: 4.5, 5
+         *                     this is REQUIRED (you know what to do)
+         *
+         *        content    - the actual review
+         *                     examples: (come on, examples? really?)
+         *                     this is REQUIRED (please be brutally honest)
+         *
+         */
+
+        // here you go:
+        review: {
+            as     : ['student', ''],
+            rating : 5,
+            content: 'Sir Arvic is like my mentor to this coding, if i never met him I would be confused of being a future developer. My knowledge and Skills are just improve by suprised yet i learned a lot from him. '
```

Comment:
Hi @Kairi-18.
**Thank you** for spending your time and effort in writing a review about me.

Before I merge this PR,
I would like to suggest some edits:

- *... my mentor to ~this~ coding*,
- *... if **I** never met him*
- *... My knowledge and **s**kills ~are~ just improve**d** by ~suprised yet~ surprise and **I** learned*...

Just edit these on your cloned repository, commit your work, then push to your fork again.